ui/raidboss/data/05-shb/raid/e3s.js

```diff
@@ -579,6 +579,14 @@
       regexCn: Regexes.startsUsing({ id: '3FEB', source: '利维亚桑', capture: false }),
       regexKo: Regexes.startsUsing({ id: '3FEB', source: '리바이어선', capture: false }),
       infoText: {
+        en: 'Front left / Back right',
+        de: 'Vorne Links / Hinten Rechts',
+        fr: 'Avant gauche / Arrière droite',
+        ja: '左前 / 右後ろ',
+        cn: '前左 / 后右',
+        ko: '↖ 앞 왼쪽 / 뒤 오른쪽 ↘',
```

Comment:
I think it would be better to ignore these arrows [here](https://github.com/quisquous/cactbot/blob/master/ui/raidboss/popup-text.js#L607) and not have a separate infoText/tts block that mostly duplicates each other.

Reply:
Ah, there has tts replacer! I didn't know, it's very clever!